bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -231,14 +262,14 @@ ByteString getStopRowByteString() {
   }
 
   RowRange getRowRange() {
-    RowSet rows = request.getRows();
+    RowSet rows = getRequest().getRows();
```

Comment:
nit, please move the encapsulation to a different PR

Reply:
It is either "getRequest()" or "request.get()". Since we are modifying the code anyway, I would rather do this now.